source/nodejs/adaptivecards-designer/src/containers/widget/widget-container-dark.css

```diff
@@ -369,6 +369,10 @@ a.ac-anchor:hover {
     border-radius:3px;
     height:38px;
     width:16px;
+    /* The arrows already have 'top: 50%' set
```

Comment:
is there a way to do this dynamically? 
and does this change work for all container size?

Reply:
Just updated and yes, it works for all container sizes